assets/js/extensions/google-analytics/index.ts

```diff
@@ -52,13 +48,6 @@ addAction(
 	namespace,
 	trackCheckoutStep( 3 )
 );
-addAction(
-	`${ actionPrefix }-checkout-set-phone-number`,
-	namespace,
-	( { step, ...rest }: { step: string; storeCart: StoreCart } ): void => {
-		trackCheckoutStep( step === 'shipping' ? 2 : 3 )( rest );
-	}
-);
```

Comment:
This was meant to track at which step we're right now, other events will continue to do so without needing this.